magprime/automated_emails.py

```diff
@@ -23,8 +23,8 @@ def __init__(self, event):
 StopsEmail('MAGFest Volunteer Food', 'volunteer_food_info.txt',
            lambda a: days_before(7, c.UBER_TAKEDOWN))
 
-StopsEmail('MAGFest Staff Suite Volunteering', 'food_interest.txt',
-           lambda a: a.requested(c.FOOD_PREP) and not a.assigned_depts)
+AutomatedEmail(Attendee, 'MAGFest Staff Suite Volunteering', 'food_interest.txt',
+           lambda a: a.requested(c.FOOD_PREP) and not a.assigned_depts, sender='chefs@magfest.org')
```

Comment:
Someone will probably need to log into our Amazon account and actually turn on the ability for this address to be the sender of our automated emails.